projects/eslint-plugin/utils/get-node-by-name-from-logical-expression.js

```diff
@@ -0,0 +1,19 @@
+const traverseLogicalExpression = require('./traverse-logical-expression');
+
+/**
+ * Return a node from the provided logical expression with the provided name.
+ * @param expression {import('@types/estree').LogicalExpression}
+ * @param name {string}
+ * @returns {import('@types/estree').Node}
+ */
+module.exports = function getNodeByNameFromLogicalExpression(expression, name) {
```

Comment:
not each node has a name. probably only identifiers have names. so perhaps this name is not the most accurate